packages/pixeloven/cli-addon-storybook/src/commands/story.ts

```diff
@@ -1,32 +1,36 @@
-import { AddonStorybookRunContext, StorybookExecutionType } from "../types";
+import { AddonStorybookToolbox, StorybookExecutionOptionTypes, StorybookExecutionType } from "../types";
 
 export default {
     alias: ["--story", "-s"],
     name: "story",
-    run: async (context: AddonStorybookRunContext) => {
-        const { parameters, pixelOven, storybook } = context;
-        if (!parameters.first) {
-            pixelOven.invalidArgument("Must provide a story run type.");
+    run: async (toolbox: AddonStorybookToolbox) => {
+        const { parameters, pixelOven, storybook } = toolbox;
+        const task = parameters.first;
+        if (!task) {
+            pixelOven.invalidArgument("Must provide a task for Storybook to run.");
```

Comment:
I should probably make this "Please provide...." to keep with the "friendly" vibe.

Reply:
haha